ebml_matroska.xml

```diff
@@ -483,9 +483,8 @@ with the top line of the top field stored first.</documentation>
     <extension type="webmproject.org" webm="1"/>
     <extension type="libmatroska" cppname="VideoStereoMode"/>
   </element>
-  <element name="AlphaMode" path="\Segment\Tracks\TrackEntry\Video\AlphaMode" id="0x53C0" type="uinteger" minver="3" default="0" maxOccurs="1">
-    <documentation lang="en" purpose="definition">Alpha Video Mode. Presence of this Element indicates that
-the BlockAdditional Element could contain Alpha data.</documentation>
+  <element name="AlphaMode" path="\Segment\Tracks\TrackEntry\Video\AlphaMode" id="0x53C0" type="uinteger" minver="3" default="0" minOccurs="1" maxOccurs="1">
+    <documentation lang="en" purpose="definition">Set to 1 if the BlockAdditional Element with BlockAddID of "1" contains Alpha data.</documentation>
```

Comment:
Thank you for having digged in how it is implemented in main browsers. [code from Google](https://github.com/chromium/chromium/blob/72ceeed2ebcd505b8d8205ed7354e862b871995e/media/formats/webm/webm_video_client.cc#L148) does not match with [WebM specs from Google](https://www.webmproject.org/docs/container/) :-p.

I understand that it is only for BlockAdditional and that it must not (?) be used for something else. Checking FFmpeg, it is not used for e.g. FFV1 RGBA. I would be more explicit about that this element is not intended for alpha mode in general despite its name (I would change it but changing a name is maybe too much complicated compared to the low usage of this element) and the issue with values of 2 and more.

Also specification of the content is not indicated (I understand from BlockAddIDType definition that we could use different formats for alpha content or something else, not only the one understood by the decoder of the format defined in CodecID).

So suggestion:

```suggestion
    <documentation lang="en" purpose="definition">Indicate if the BlockAdditional Element with BlockAddID of 1 contains Alpha data conforming to the specification indicated by CodecID for alpha additional data passed to the codec. Undefined values SHOULD NOT be used as the behavior of known implementations is different (considered either as 0 or 1).</documentation>
    <restriction>
      <enum value="0" label="The BlockAdditional Element with BlockAddID of 1 does not exist or SHOULD NOT be considered as containing such data."/>
      <enum value="1" label="The BlockAdditional Element with BlockAddID of 1 contains such data."/>
    </restriction>
```
(BlockAddID is an uinteger so I would not use the quote for it, I intentionally removed quotes around its value).